Classes/View Controllers/NavigationController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  NavigationController.swift
+//  Freetime
+//
+//  Created by Nathan Tannar on 2018-12-16.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+
+final class NavigationController: UINavigationController, ThemeChangeListener {
```

Comment:
Suggestion: What about calling `NavigationController`, `GHNavigationController`? (GH for GitHawk)

Reply:
I didn't see GH[ClassName] anywhere else so I didn't think this was a convention that was being followed